man/meson.build

```diff
@@ -218,7 +218,7 @@ configure_file(
 ############################################################
 
 update_dbus_docs = custom_target(
-        'update-dbus-docs',
+        'update-dbus-docs-impl',
```

Comment:
"conflicted" → "conflicting".

Hmm, what is it conflicting with? I would prefer to keep the existing user-visible name.